test/unit/app/controllers/transactions/pending-tx-tracker-test.js

```diff
@@ -0,0 +1,616 @@
+import sinon from 'sinon'
+import { strict as assert } from 'assert'
+import PendingTransactionTracker from '../../../../../app/scripts/controllers/transactions/pending-tx-tracker'
+
+describe('PendingTransactionTracker', function () {
```

Comment:
```suggestion
describe('PendingTransactionTracker', function () {

after(function () {
  sinon.restore()
}
```

Reply:
I don't _think_ this is necessary where all of the spies are scoped to the function—do we still need to call this?

Reply:
You're right, just saw it as a habit to make.

Reply:
Gotcha, yeah I think it might be easier if we don't have any global/cross-test mocks 😅